src/index.js

```diff
@@ -0,0 +1,235 @@
+import './style.css';
+import './comment.css';
+import Logo from './dragon.jpeg';
+
+const myLogo = new Image();
+myLogo.src = Logo;
+const logoDiv = document.querySelector('#logo-div');
+logoDiv.appendChild(myLogo);
+
+const baseUri = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const appId = 'K5LEyqREMBDZLL96ZFuw';
+
+const addLike = async (itemId) => {
+  const response = await fetch(`${baseUri}${appId}/likes`, {
+    method: 'POST',
+    body: JSON.stringify(
+      {
+        item_id: itemId,
+      },
+    ),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  response.json().then((json) => {
+    console.log(json);
```

Comment:
- [ ] Let's remove `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.